java/core/src/java/org/apache/orc/impl/HadoopShimsCurrent.java

```diff
@@ -33,31 +34,62 @@
  */
 public class HadoopShimsCurrent implements HadoopShims {
 
-  private static class DirectDecompressWrapper implements DirectDecompressor {
-    private final org.apache.hadoop.io.compress.DirectDecompressor root;
+  private static class SnappyDirectDecompressWrapper implements DirectDecompressor {
```

Comment:
Is this change required? I don't see any change that require having separate Snappy and Zlib wrappers.